build-system/tasks/update-packages.js

```diff
@@ -110,15 +110,13 @@ function patchRegisterElement() {
 }
 
 /**
- * Patch @ampproject/worker-dom/dist/index.safe.js with ES6 export.
+ * Closure Compiler doesn't recognize .mjs extension yet, so copy the file to
+ * have a .js file extension.
  */
 function patchWorkerDom() {
-  replaceInFile(
-      'node_modules/@ampproject/worker-dom/dist/index.safe.js',
-      'node_modules/@ampproject/worker-dom/dist/index.safe.patched.js',
-      // Replace local var with an ES6 export.
-      'var MainThread=function(R){',
-      'export const MainThread=function(R){');
+  fs.copyFileSync(
+      'node_modules/@ampproject/worker-dom/dist/unminified.index.safe.mjs',
+      'node_modules/@ampproject/worker-dom/dist/unminified.index.safe.mjs.js');
```

Comment:
Can likely just replace the extension here.

```suggestion
      'node_modules/@ampproject/worker-dom/dist/unminified.index.safe.js');
```

Reply:
I didn't want to overwrite the existing file to avoid a source of confusion.